CMakeLists.txt

```diff
@@ -103,8 +103,15 @@ if(CMAKE_WANT_UFE_BUILD)
 endif()
 
 if(DEFINED QT_LOCATION)
+    if(NOT DEFINED QT_VERSION)
```

Comment:
Normally we'll automatically set the Qt version based on the Maya you are building for. But allow using a specific version of Qt by setting QT_VERSION.